docs/farming.md

```diff
@@ -52,7 +62,17 @@ After this, simply repeat the step you prompted for (step 4 or 6). This time, cl
 2. Open your favourite terminal, and go to the folder where you download the executables
 3. This will start the node (replace `INSERT_YOUR_ID` with a nickname you choose):
 ```
-subspace-node-x86_64-*-snapshot --chain testnet --wasm-execution compiled --execution wasm --bootnodes "/dns/farm-rpc.subspace.network/tcp/30333/p2p/12D3KooWPjMZuSYj35ehced2MTJFf95upwpHKgKUrFRfHwohzJXr" --rpc-cors all --rpc-methods unsafe --ws-external --validator --telemetry-url "wss://telemetry.polkadot.io/submit/ 1" --name INSERT_YOUR_ID
+subspace-node-x86_64-*-snapshot \
```

Comment:
The reason it was written as one line for Windows is because there were some issues with this style at least in some shells. I guess Linux example was just copy-pasted back for unified look.

Reply:
can I keep the original for windows, but update linux/mac then?

Reply:
Yes, that'll be fine